src/index.jsx

```diff
@@ -6,7 +6,9 @@ import store from './Mainstore/Store';
 
 ReactDOM.render(
   <Provider store={store}>
-    <App />
+    <div>
```

Comment:
May I know why you had to change this file by enclose `<App />` under the `<div>` element.

Reply:
This inside the div tag

Reply:
@Emmanuelaaron I also don't understand why you had to make that change. Is it necessary? If not, remove it.